flyway-commandline/pom.xml

```diff
@@ -80,13 +80,10 @@
             <groupId>org.xerial</groupId>
             <artifactId>sqlite-jdbc</artifactId>
         </dependency>
-
```

Comment:
Please don't mess with whitespace lines like these. They are placeholders for keeping line numbers in sync between the community, pro and enterprise editions.